com.unity.multiplayer.mlapi/Runtime/Serialization/BitWriter.cs

```diff
@@ -920,8 +920,11 @@ public void WriteBytes(byte[] buffer, long targetSize)
         public void ReadAndWrite(BitReader sourceReader, long targetSize)
         {
             long TargetSize = targetSize;
-            long LargeInt64Blocks = (long)((float)TargetSize * 0.125f);
-
+            long LargeInt64Blocks = TargetSize >> 3;
+            if(targetSize > 131072 || targetSize < 0)
```

Comment:
I don't like magic numbers, is this the same magic number as above? you could use the same `const` variable on both sides.